src/c/runtime/advice_chrome_tracing.cpp

```diff
@@ -207,6 +238,10 @@ int advice_chrome_tracing_t::cannonicalize_perfflow_options()
     {
         m_perfflow_options["log-enable"] = "True";
     }
+    if (m_perfflow_options.find("cpu_mem_usage") == m_perfflow_options.end())
+    {
+        m_perfflow_options["cpu_mem_usage"] = "False";
```

Comment:
If we find `cpu_mem_usage` in PERFFLOW_OPTIONS, shouldn't the value of `m_perfflow_options["cpu_mem_usage"]` be True?

Reply:
I believe we discussed this would be False by default, should I make it True by default?